### DISCUSSION TITLE
auth_token

### DISCUSSION THREAD

**Discussion by daveanvit** [Sunbird-Ed/Community#149] [Installation] [answered]
I had deployed jobs with invalid core_vault_sunbird_api_auth_token, however I had corrected it, which jobs do i need to redeploy?

**Comment by saiakhil46** [comment, marked as answer]
@daveanvit Redeploy below services.

Core jobs:
lms, learner, knowledge-mw, content, player, report-service, discusion-mw-service, ml-analytics-service

Datapipeline jobs:
redeploy all datapipeline flink jobs, eddataporducts, coredataproducts

learning platform jobs:
learning-service, redeploy all knowledgeplatform flink jobs, yarn jobs

**Reply by daveanvit** [reply]
Thanks @saiakhil46 this worked
